finalPrep/README.md

```diff
@@ -18,37 +18,43 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [X] I have refactored my `section1` exercises to the best of my ability
+- [X] I have refactored my `section2` exercises to the best of my ability
+- [X] I have refactored my `section3` exercises to the best of my ability
+- [X] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [X] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [X] Study/Project work time
+- [X] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1440" alt="Screen Shot 2021-08-07 at 6 43 05 PM" src="https://user-images.githubusercontent.com/78453792/128618929-55288abf-f61d-44d1-9c2a-f878825de997.png">
+
+- `Add Week 2 Screenshot Here`<img width="1440" alt="Screen Shot 2021-08-07 at 6 43 09 PM" src="https://user-images.githubusercontent.com/78453792/128618945-422fbfeb-d498-4a1b-ab40-fdc6d71cef64.png">
+
+- `Add Week 3 Screenshot Here` <img width="1440" alt="Screen Shot 2021-08-07 at 6 43 13 PM" src="https://user-images.githubusercontent.com/78453792/128618955-c2ca1ede-0085-4655-a507-cb3f06172e65.png">
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] Add link to your gist here: <a href="https://gist.github.com/cagallo/e7a92a46409f78a4d81eb88d94d91f9c">Mentorship DTR Prep</a>
```

Comment:
Good job with the intro to your mentor. You may think about including a detail or so about what excites you about starting Turing, or what makes you nervous- this may help your mentor identify ares where they could be of support.